08-secret-managment-and-ingress-controller.md

```diff
@@ -99,7 +99,7 @@ Previously you have configured [workload prerequisites](./08-workload-prerequisi
    :warning: Deploying the traefik `traefik.yaml` file unmodified from this repo will be deploying your workload to take dependencies on a public container registry. This is generally okay for learning/testing, but not suitable for production. Before going to production, ensure _all_ image references are from _your_ container registry or another that you feel confident relying on.
 
    ```bash
-   kubectl create -f https://raw.githubusercontent.com/mspnp/aks-secure-baseline/main/workload/traefik-03.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_03
+   kubectl create -f https://raw.githubusercontent.com/<your user>/aks-secure-baseline/main/workload/traefik-03.yaml --context $AKS_CLUSTER_NAME_BU0001A0042_03
```

Comment:
Please let's create an ENV VAR at the beginning for this URL and use it from all the sections required.

RAW_GITHUB_CONTENT_FORK=https://raw.githubusercontent.com/${GITHUB_USER_NAME}/aks-secure-baseline/main

Reply:
done